src/main/java/org/spongepowered/common/mixin/plugin/tileentityactivation/TileEntityActivation.java

```diff
@@ -227,13 +227,14 @@ public static boolean checkIfActive(TileEntity tileEntity) {
 
     public static void addTileEntityToConfig(World world, SpongeTileEntityType type) {
         SpongeConfig<? extends GeneralConfigBase> config = ((IMixinWorldServer) world).getActiveConfig();
-        if (config == null || type == null || !config.getConfig().getTileEntityActivationRange().autoPopulateData()) {
+        SpongeConfig<? extends GeneralConfigBase> globalConfig = SpongeImpl.getGlobalConfig();
+        if (config == null || globalConfig == null || type == null || !config.getConfig().getTileEntityActivationRange().autoPopulateData()) {
             return;
         }
 
         boolean requiresSave = false;
         final String tileModId = type.getModId().toLowerCase();
-        TileEntityActivationCategory activationCategory = config.getConfig().getTileEntityActivationRange();
+        TileEntityActivationCategory activationCategory = globalConfig.getConfig().getTileEntityActivationRange();
         TileEntityActivationModCategory tileEntityMod = activationCategory.getModList().get(tileModId);
         int defaultRange = activationCategory.getDefaultBlockRange();
```

Comment:
Same as other places, the value should be read from the "active" `config` instance.

Reply:
Nevermind, doesn't apply here.